test/functional/ex_cmds/wviminfo_spec.lua

```diff
@@ -0,0 +1,54 @@
+local helpers, lfs, os =
+  require('test.functional.helpers'), require('lfs'), require('os')
+local clear, execute, eq, neq, spawn, nvim_prog, set_session =
+  helpers.clear, helpers.execute, helpers.eq, helpers.neq, helpers.spawn,
+  helpers.nvim_prog, helpers.set_session
+
+describe(':wviminfo', function()
+  local file = './foo'
+  before_each(function()
+    clear()
+    os.remove(file)
+  end)
+  after_each(function() os.remove(file) end)
```

Comment:
This is redundant with `before_each` since it's doing the same thing, and `before_each` will run before each `it` test.